app/services/github_client/user.rb

```diff
@@ -7,10 +7,25 @@ def initialize(username)
     end
 
     def pull_request_events
-      events = (1..10).map do |page|
+      events = retrieve_events
+
+      events.flatten.select { |event| event[:type] == 'PullRequestEvent' }
+    rescue URI::InvalidURIError => exception
+      ExceptionHunter.track(exception)
+    end
+
+    private
+
+    def older_than(created_at)
+      DateTime.parse(created_at) < ENV.fetch('SCHEDULE_EXTERNAL_PULL_REQUESTS', 1).to_i.days.ago
+    end
+
+    def retrieve_events
+      (1..10).map do |page|
```

Comment:
WDYT about using `flat_map` here, so we can just do `events.select { |event| event[:type] == 'PullRequestEvent' }` in `#pull_request_events`?